doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1,16 +1,13 @@
 .. _ncs_release_notes_changelog:
 
-Changelog for |NCS| v3.0.99
-###########################
+Changelog for |NCS| v3.1.0-preview1
+###################################
 
 .. contents::
    :local:
    :depth: 2
 
-The most relevant changes that are present on the main branch of the |NCS|, as compared to the latest official release, are tracked in this file.
-
-.. note::
-   This file is a work in progress and might not cover all relevant changes.
+This changelog reflects the most relevant changes from the latest official release.
```

Comment:
Line 158: spacing issue.
Line 214: Updated the application configurations..
Line 283: Could have it under 'Deprecated:'
Line 503: a need to